src/qt/guiutil.cpp

```diff
@@ -1007,4 +1009,15 @@ void ShowModalDialogAsynchronously(QDialog* dialog)
     dialog->show();
 }
 
+void bringToFrontWorkaroundForWayland(QWidget* w)
+{
+    if (QGuiApplication::platformName() == "wayland") {
+        auto eFlags = w->windowFlags();
```

Comment:
The variable name looks confusing to me. Why not just "flags"?

Reply:
Yeah, I'll change it.